src/multiqc.sh

```
@@ -119,13 +77,14 @@ main() {
     # Load the docker image and then run it
     docker load -i MultiQC.tar.gz
     MultiQC_image=$(docker images --format="{{.Repository}} {{.ID}}" | grep multiqc | cut -d' ' -f2)
-    docker run -v /home/dnanexus:/egg $MultiQC_image /egg/"$folder_name" -c /egg/config.yaml -n /egg/${outdir}/$report_name
+    docker run -v /home/dnanexus:/egg -w /egg $MultiQC_image multiqc "$folder_name" -c config.yaml
 
     echo "Uploading the config file, html report and a folder of data files"
+    mv multiqc_data ${outdir}/
     # Move the config file to the multiqc data output folder. This was created by running multiqc
     mv config.yaml ${outdir}/$multiqc_config_file_name
```

Shellcheck (0.10.0)

[warning] multiqc_config_file_name is referenced but not assigned.

(SC2154)